EncodingStreamingHighAvailability/HighAvailability.InstanceHealth/Startup.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+
+[assembly: FunctionsStartup(typeof(HighAvailability.InstanceHealth.Startup))]
+
+namespace HighAvailability.InstanceHealth
```

Comment:
can we change the root folder in Github to use the name HighAvailability at the front and not at the end? That would make it easier to see.

Reply:
done